Improved design of website.

Areebah03 opened the pull request:
Fixed #360

# Description

I have improved the design of the template. Added animation effects, logo , contents.

Fixes # (360)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions

**Test Configuration**:

- Operating system:
- Version:
- Text-editors used:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings


# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
![Screenshot 2022-03-07 161931](https://user-images.githubusercontent.com/85051124/157017259-1c5cf96e-e4d2-4cdf-9ebc-7aa99e5f3961.png)
![Screenshot 2022-03-07 161904](https://user-images.githubusercontent.com/85051124/157017333-293cf9d7-0ea7-4df2-a599-07f54d4ffe2c.png)

arpit456jain commented:
![image](https://user-images.githubusercontent.com/55352601/157209547-72c9d14c-1be8-4bc9-ad71-80a841d39cfb.png)

VidhiBhatt01 commented:
> ![image](https://user-images.githubusercontent.com/55352601/157209547-72c9d14c-1be8-4bc9-ad71-80a841d39cfb.png)

@Areebah03 fix this please and also add before/ after image in the description

Areebah03 commented:
Fixed #360
Fixed #376 
  
# Description

I have improved the design of the template. I have fixed service section and also modified it. I have improve the hover effect of links and modified button.

Fixes # (360)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions

**Test Configuration**:

- Operating system:
- Version:
- Text-editors used:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings


# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
![Screenshot 2022-03-08 194845](https://user-images.githubusercontent.com/85051124/157258339-c2662de1-9547-4b11-805f-034431aa401b.png)
![Screenshot 2022-03-08 194910](https://user-images.githubusercontent.com/85051124/157258390-e8ec8b98-32eb-4545-a718-08f03db3a1be.png)
